vespalib/src/vespa/vespalib/stllike/asciistream.cpp

```diff
@@ -235,6 +235,17 @@ asciistream & asciistream::operator >> (char & v)
     return *this;
 }
 
+asciistream & asciistream::operator >> (signed char & v)
```

Comment:
asciistream_test.cpp should be updated with test case for this.

Reply:
Yes, will do.